query-engine/query-engine-wasm/build.sh

```diff
@@ -38,13 +41,15 @@ WASM_OPT_ARGS=(
     "--dae-optimizing"                    # removes arguments to calls in an lto-like manner
     "--remove-unused-names"               # removes names from location that are never branched to
     "--rse"                               # removes redundant local.sets
-    "--gsi"                               # global struct inference, to optimize constant values
-    "--gufa-optimizing"                   # optimize the entire program using type monomorphization
```

Comment:
Was `--gufa-optimizing` a performance optimization that increased size instead?

Reply:
I have the same question

Reply:
Yes, but this belongs to the next PR, moving it there.